vendor-overwrites/colorpicker/colorconverter.js

```diff
@@ -42,6 +42,41 @@ const colorConverter = (() => {
     }
   }
 
+  // Copied from _hexcolor() in parserlib.js
+  function validateHex(color) {
+    return /^#[a-f\d]+$/i.test(color) && [4, 5, 7, 9].some(n => color.length === n);
+  }
+
+  function validateRGB(nums) {
+    const isPercentage = nums[0].endsWith('%');
+    const valid = isPercentage ? validatePercentage : validateNum;
+    return nums.slice(0, 3).every(valid);
+  }
+
+  function validatePercentage(s) {
+    const match = s.match(/^(\d+|\d*\.\d+)%$/);
+    return match && Number(match[1]) >= 0 && Number(match[1]) <= 100;
+  }
+
+  function validateNum(s) {
+    return Number(s) >= 0 && Number(s) <= 255;
```

Comment:
The [spec](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Syntax_2) says the value must be an integer. It gets rounded off when saved, but do we need to inform the developer?

Reply:
:+1: I guess it should be changed to
```js
return /^\d+$/.test(s) && ...
```